Octokit/Models/Response/Enterprise/PreReceiveEnvironmentDownload.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    /// <summary>
+    /// Describes the current download state of a pre-receive environment image.
+    /// </summary>
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PreReceiveEnvironmentDownload
```

Comment:
response class needs the 2 ctors mentioned previously

Reply:
Updated